frontend/src/metabase/visualizations/echarts/pie/option.ts

```diff
@@ -0,0 +1,44 @@
+import type { EChartsOption, RegisteredSeriesOption } from "echarts";
+
+import type {
+  ComputedVisualizationSettings,
+  RenderingContext,
+} from "metabase/visualizations/types";
+
+import type { PieChartModel } from "./model/types";
+
+// static, non-computed values
+const SUNBURST_SERIES_OPTIONS: RegisteredSeriesOption["sunburst"] = {
+  type: "sunburst",
+  nodeClick: false,
+  sort: undefined,
+  emphasis: { focus: "none" },
+  label: {
+    rotate: 0,
+    overflow: "none",
+    lineHeight: 50, // TODO confirm
```

Comment:
Later when polishing will get the exact values from existing chart and confirm with designers